catalog/transformer/catalog-transformer-mgmp/src/main/java/org/codice/alliance/catalog/transformer/mgmp/MgmpTransformer.java

```diff
@@ -486,6 +486,16 @@ private void addMetacardDataQuality(MetacardImpl metacard) {
         try (InputStream inputStream = getSourceInputStream()) {
             DocumentBuilderFactory domFactory = DocumentBuilderFactory.newInstance();
             domFactory.setNamespaceAware(false);
+            try {
+                domFactory.setFeature(
+                        "http://apache.org/xml/features/nonvalidating/load-dtd-grammar",
+                        false);
+                domFactory.setFeature(
+                        "http://apache.org/xml/features/nonvalidating/load-external-dtd",
+                        false);
```

Comment:
Per https://xerces.apache.org/xerces-j/features.html this seems to indicate that these features can only be disabled if validation is disabled. Does that mean we are not validating the incoming data and just handing it over to the parser?